### PR TITLE
fix: update compute class documentation to be in line with sandbox/pro current state

### PR DESCRIPTION
**Pull request by keyallis**
Update with naming scheme of `free` and `standard` with `free` as the default. Remove references of byoc compute class offerings.

### CODE REVIEW

File: docs/30-production/05-acorn-sizing.md [hunk 1]
```diff
@@ -104,19 +101,19 @@ When describing an Acorn deployment in an Acornfile, you can specify the compute
...
 services: mariadb: {
-    class: "m5"
+    class: "standard"
     image: "ghcr.io/acorn-io/mariadb:v10.#.#-#"
     autoUpgrade: true
 }
 
 acorns: app: {
-    class: "c5"
+    class: "free"
```
Comment (cloudnautique) [inline docs/30-production/05-acorn-sizing.md]: Should we be showing this as an example? Won't this now be deleted every so many hours?

Comment (keyallis) [reply docs/30-production/05-acorn-sizing.md]: The docs go through the process of leveraging two compute classes to show how to use update to set a new class on an existing object. To do that we would need to use both compute classes that we have available. If you end up wanting to further restructure or reduce down the options we have we could go that route, but I just went with minimal changes to try and retrofit existing docs for the new use case

Comment (cjellick) [reply docs/30-production/05-acorn-sizing.md]: While mixing and matching is technically possible, I would prefer we not encourage it in the docs. Can you change this to standard and then just implify the text you're adding to something like 

Note: if you don't specify a compute class, the default compute class will be used, same as on the command line.

**Review by cjellick — CHANGES_REQUESTED** [review]
see my inline comment. one small tweak

**Review by cjellick — APPROVED** [review]

**Review by cloudnautique — APPROVED** [review]

**Pull request merged**